src/formatter/date.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  DateFromString,
+  DateFromTimestamp,
+  UtcOnlyIsoDateFromString
+} from "@pagopa/ts-commons/lib/dates";
+import * as E from "fp-ts/Either";
+import { pipe } from "fp-ts/function";
+import * as t from "io-ts";
+
+const invalidDateStringError = (): Error => new Error("Invalid dateString");
+
+export const dateStringToUtcFormat = (
+  dateString: string
+): E.Either<Error, string> =>
+  pipe(
+    dateString,
+    DateFromString.decode,
+    E.bimap(invalidDateStringError, date => date.toUTCString())
+  );
+
+export const isoToUtcFormat = (isoString: string): E.Either<Error, string> =>
+  pipe(
+    isoString,
+    UtcOnlyIsoDateFromString.decode,
+    E.bimap(
+      () => new Error("Invalid isoString"),
+      date => date.toUTCString()
+    )
+  );
+
+export const dateStringToIsoFormat = (
+  dateString: string
+): E.Either<Error, string> =>
+  pipe(
+    dateString,
+    DateFromString.decode,
+    E.bimap(invalidDateStringError, date => date.toISOString())
+  );
+
+export const dateStringToTimeStampFormat = (
+  dateString: string
+): E.Either<Error, number> =>
+  pipe(
+    dateString,
+    DateFromString.decode,
+    E.bimap(invalidDateStringError, date => date.getTime())
+  );
+
+export const dateStringFromTimestampFormat = (
+  ts: number
+): E.Either<Error, string> =>
+  pipe(
+    ts,
+    DateFromTimestamp.decode,
```

Comment:
The DateFromTimestamp decoder should only accept valid timestamps such as 1702944000000 as input. However, even if it receives a random number it does not give an error.